uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader.php

```diff
@@ -38,6 +38,8 @@ class CRM_HRCore_Upgrader extends CRM_HRCore_Upgrader_Base {
   use CRM_HRCore_Upgrader_Steps_1028;
   use CRM_HRCore_Upgrader_Steps_1029;
   use CRM_HRCore_Upgrader_Steps_1030;
+  use CRM_HRCore_Upgrader_Steps_1031;
```

Comment:
You added two upgraders here (`1031` and `1032`). Is there any reason for that? 🤷‍♂️